tmt/steps/report/junit/templates/_base.xml.j2

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" ?>
+{% block content %}
+<testsuites disabled="0" errors="{{ RESULTS.errored | length }}" failures="{{ RESULTS.failed | length }}" tests="{{ RESULTS | length }}" time="{{ RESULTS.duration }}">
+    {% block testsuites %}
+    <testsuite name="{{ PLAN.name | trim | e }}" disabled="0" errors="{{ RESULTS.errored | length }}" failures="{{ RESULTS.failed | length }}" skipped="{{ RESULTS.skipped | length }}" tests="{{ RESULTS | length }}" time="{{ RESULTS.duration }}">
+        {% block testcases %}
+            {% for result in RESULTS %}
+                {% set main_log = result.log | first | read_log %}
+                {% set log_failures = main_log | failures | e %}
+                {% set test_duration = result.duration | duration_to_seconds %}
+
+                <testcase name="{{ result.name | e }}" {% if test_duration > 0 %}time="{{ test_duration }}"{% endif %}>
+                    {% if result.result.value == 'error' or result.result.value == 'warn' %}
+                        <error type="error" message="{{ result.result.value | e }}">{{ log_failures }}</error>
+                    {% elif result.result.value == 'fail' %}
+                        <failure type="failure" message="{{ result.result.value | e }}">{{ log_failures }}</failure>
+                    {% elif result.result.value == 'info' %}
+                        <skipped type="skipped" message="{{ result.result.value | e }}">{{ log_failures }}</skipped>
+                    {% endif %}
+
+                    {% if INCLUDE_OUTPUT_LOG and main_log %}
+                        <system-out>{{ main_log | e }}</system-out>
+                    {% endif %}
+
+                    {# Optionally add the result properties #}
+                    {% if result.properties is defined %}
+                        {% with properties=result.properties %}
+                            {% include "includes/_properties.xml.j2" %}
```

Comment:
since you already have it here why not add it for the testsuites as well?

also, properties are missing from the xsd so it would fail validation if someone actually used it before the polarion part gets merged

Reply:
I think the `testsuites` properties should be probably part of the `ResultsContext` in a very similar way as for `ResultWrapper`. I would like to implement it probably as a part of #3166 so this PR will make it into v1.36.